hoot-core/src/main/cpp/hoot/core/io/OsmXmlReader.cpp

```diff
@@ -106,13 +106,21 @@ void OsmXmlReader::_parseTimeStamp(const QXmlAttributes &attributes)
 
 void OsmXmlReader::_createNode(const QXmlAttributes& attributes)
 {
+  _element.reset();
+
```

Comment:
When ignoring elements with duplicate IDs, the `_element` pointer needed to be reset so that it wasn't modified after it shouldn't.